ld/testsuite/ld-riscv-elf/relocid.d

```diff
@@ -0,0 +1,25 @@
+#source: relocid.s
+#as: -march=rv32ixcvhwlp -mabi=ilp32
+#ld: -melf32lriscv -Ttext 0x8000
```

Comment:
If we want to test that specific relocations are emitted, wouldn't we want to not run the linker?

Reply:
Yep you are right, we do not want to run the linker - I'll fix this.